srcpkgs/lemoa/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'lemoa'
+pkgname=lemoa
+version=0.2.0
+revision=1
+build_style=meson
+build_helper=rust
+hostmakedepends="pkg-config rust cargo desktop-file-utils glib-devel gtk-update-icon-cache"
+makedepends="rust-std gtk4-devel libadwaita-devel"
+short_desc="Gtk client for Lemmy"
+maintainer="Bnyro <bnyro@tutanota.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/lemmygtk/lemoa"
+distfiles="https://github.com/lemmygtk/lemoa/releases/download/v${version}/lemoa-${version}.tar.xz"
+checksum=602974f62557f2476e0b7d081d9c727fec748e833d8c6e4cc112e9208c09a7c2
+
+pre_build() {
+	if [ "$CROSS_BUILD" ]; then
+		vsed -i src/meson.build \
+			-e "s%rust_target /%'${RUST_TARGET}' / rust_target /%"
+	fi
```

Comment:
This should probably be `post_patch` or `post_extract`, meson will reconfigure the build if you change the meson.build file. I guess meson is smart enough to reuse the previous configuration, but its cleaner to patch prior to configuring.